poly-markdown.el

```diff
@@ -66,10 +66,11 @@
   :tail-matcher (pm-make-text-property-matcher 'markdown-yaml-metadata-end))
 
 ;; allow extra . before language name https://github.com/polymode/polymode/issues/296
+;; allow extra = before language name https://github.com/polymode/poly-markdown/issues/22
 (define-auto-innermode poly-markdown-fenced-code-innermode poly-markdown-root-innermode
-  :head-matcher (cons "^[ \t]*\\(```[ \t]*{?[[:alpha:].].*\n\\)" 1)
+  :head-matcher (cons "^[ \t]*\\(```[ \t]*{?[[:alpha:].=].*\n\\)" 1)
   :tail-matcher (cons "^[ \t]*\\(```\\)[ \t]*$" 1)
-  :mode-matcher (cons "```[ \t]*{?\\.?\\(?:lang *= *\\)?\\([^ \t\n;=,}]+\\)" 1))
+  :mode-matcher (cons "```[ \t]*{?\\[\.=]?\\(?:lang *= *\\)?\\([^ \t\n;=,}]+\\)" 1))
```

Comment:
The regexp here does not look right. It should be `{?[.=]?` I think.

Reply:
Fixed.

I wonder how `\\.` worked, as that would match a slash `\` and any character.